.github/workflows/node.js.yml

```diff
@@ -31,4 +31,5 @@ jobs:
           cache-dependency-path: typescript-id-allocator/package-lock.json
       - run: cargo install wasm-snip
       - run: npm run build --verbose
+      - run: npm run eslint
```

Comment:
add format, see if both can run in verbose mode

Reply:
Looks like both eslint and prettier are "verbose" by default. Adding github actions verbosity just shows it setting up the tools and running them, no impact on the resulting error readouts.